linux_os/guide/system/network/network_ssl/only_allow_specific_certs/tests/ocp4/e2e.yml

```diff
@@ -0,0 +1,2 @@
+---
+default_result: MANUAL
```

Comment:
We could use an assertion file to capture the entire posture of the BSI profile in a single place.

We should be able to use the one from CI results: 

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/ComplianceAsCode_content/13121/pull-ci-ComplianceAsCode-content-master-4.16-e2e-aws-rhcos4-bsi/1895431334566825984#1:build-log.txt%3A1111

Reply:
@rhmdnd  shall I add this to this PR or to a seperate one?